metabase/metabase
# Fix and reproduce #20499: The term "SQL query" used in DBs that don't support SQL

### Status
PENDING REVIEW

### What does this PR accomplish?
- Fixes #20499 
- Reproduces #20499 
- Updates all tests that were using the term "SQL query"

### How to test?
_Make sure you have Mongo DB connected (or GA, or Druid)_
1. Visit a home page
2. Click on the "New"
3. Popover should display "Native query" instead of "SQL query"

_For the sample database and every other DB that supports SQL_
1. Visit a home page
2. Click on the "New"
3. Popover should display "SQL query"
